src/main/java/sg/ncl/MainController.java

```diff
@@ -3168,6 +3169,45 @@ private String unfreezeUser(final User2 user, RedirectAttributes redirectAttribu
         }
     }
 
+    @RequestMapping("/admin/users/remove/{userId}")
```

Comment:
I followed the same format as in other user/team management functions in MainController. So should I change to this one?